alvr/client_openxr/src/stream.rs

```diff
@@ -428,21 +436,29 @@ fn stream_input_loop(
             &reference_space,
             tracker_time,
             &interaction_ctx.hands_interaction[0],
-            &mut last_hand_positions[0],
+            &mut last_controller_poses[0],
+            &mut last_palm_poses[0],
         );
         let (right_hand_motion, right_hand_skeleton) = crate::interaction::get_hand_data(
             &xr_ctx.session,
             &reference_space,
             tracker_time,
             &interaction_ctx.hands_interaction[1],
-            &mut last_hand_positions[1],
+            &mut last_controller_poses[1],
+            &mut last_palm_poses[1],
         );
 
-        if let Some(motion) = left_hand_motion {
-            device_motions.push((*HAND_LEFT_ID, motion));
+        // Note: When multimodal input is enabled, we are sure that when free hands are used
+        // (not holding controllers) the controller data is None.
```

Comment:
huh? isn't it the other way around?

Reply:
it's not the other way around, it's just not correct... i'll fix it

Reply:
Nevermind, it's already correct.

Reply:
but isn't that the whole purpose of multimodal? that we get data for both controllers and hands?

Reply:
aaah, we're not doing detached yet, makes sense